evap/staff/templates/staff_semester_view.html

```diff
@@ -26,6 +26,7 @@ <h3>
                 <div class="ml-auto">
                     {% if not semester.is_archived %}
                         <a href="{% url 'staff:semester_todo' semester.id %}" class="btn btn-sm btn-light">{% trans 'Todo' %}</a>
+                        <a href="{% url 'staff:semester_grade_reminder' semester.id %}" class="btn btn-sm btn-light">{% trans 'Grade Publish Reminder' %}</a>
```

Comment:
`Grade publish reminder` again, please

Reply:
and please add an `ml-2` to the class attribute to make the spacing between the buttons consistent